Eniko/index.js

```diff
@@ -0,0 +1,15 @@
+let a = 1;
+let b = 2;
+
+
+//A
+if (a > b) {
+    console.log(a, "is the bigger number");
+} else {
+    console.log(b, "is the bigger number");
+}
+
+//B
+console.log(a === b);
+//C - i am not sure about this rule
+console.log(a > b);
```

Comment:
Just try to add if a is bigger than b then print true in console otherwise(if not) then false